### DISCUSSION TITLE
Guess X countries

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#667] [Galaxy Trivia] [answered]
Below, 10 countries have been broken into chunks of letters.

These chunks have been mixed up, no chunk is used twice and all chunks are used.

```
EZU ITZ ZIL ELA BRA GI IA FI
PAN MBA BEL AND BER ZI NL BO
CAM VEN DIA AND     UM SW
BWE MEX ERL ICO     JA LI
```

Find out what they are and answer with their country's flag.

**Comment by gotthejuicee** [comment]
The countries are:

Brazil 🇧🇷
Belgium 🇧🇪
Zimbabwe 🇿🇼
Italy 🇮🇹
Algeria 🇩🇿
Finland 🇫🇮
Panama 🇵🇦
Netherlands 🇳🇱
Cambodia 🇰🇭
Swaziland (Eswatini) 🇸🇿
Note: The chunk "UM" corresponds to the country Eswatini, which was formerly known as Swaziland.

**Comment by peterbaccaro** [comment, marked as answer]
🇧🇪  BEL-GI-UM = BELGIUM
🇲🇽  MEX-ICO = MEXICO
🇧🇷  BRA-ZIL = BRAZIL
🇯🇵  JA-PAN = JAPAN
🇰🇭  CAM-BO-DIA = CAMBODIA
🇨🇭  SW-ITZ-ERL-AND = SWITZERLAND
🇿🇼  ZI-MBA-BWE = ZIMBABWE
🇻🇪  VEN-EZU-ELA = VENEZUELA
🇫🇮  FI-NL-AND = FINLAND
🇱🇷  LI-BER-IA = LIBERIA

**Comment by SANKARAMDAS** [comment]
br BRA-ZIL   
jp JA-PAN   
mx MEX-ICO   
fi FI-NL-AND  
zw ZI-MBA-BWE   
be BEL-GI-UM  
sz SW-ITZ -ERL-AND    
ve VEN-EZU-ELA    
zw ZI-MBA-BWE
lr LI-BER-IA         
kh CAM-BO-DIA

**Reply by peterbaccaro** [reply]
Referee!!  This answer is actually wrong as the question asks for the flags and this answer does not include any flags.

**Reply by ParanoidUser** [reply]
🙈 Oopsie daisy! Looks like my eagle eyes missed those cheeky little emojis. Revised towards the prosecution. 🧑‍⚖️

**Reply by peterbaccaro** [reply]
🤣🤣🤣
